src/Microsoft.DotNet.CodeAnalysis/project.json

```diff
@@ -0,0 +1,23 @@
+{
+  "dependencies": {
+    "Microsoft.Net.Compilers.NetCore": "1.2.0-beta1-20160202-02",
```

Comment:
I think this needs to be the netcore one. We are importing the other one in the 4.5 project.